app/assets/stylesheets/application/common.scss

```diff
@@ -266,3 +268,66 @@ header .container-fluid nav.navbar li.dropdown {
     line-height: 1.5;
   }
 }
+.body-content {
+  width: 92%;
+  margin-top: 5%;
+  margin-right: auto;
+  margin-left: auto;
+}
+.body-content-element {
+  padding-bottom: 4%;
+  border-radius: 5px;
+  margin-bottom: 3%;
+  box-shadow: 0 0 20px 0 rgba(0,0,0,0.2),0 5px 5px 0 rgba(0,0,0,0.24);
+}
+.content-title {
+  padding: 1% 0%;
+  margin-bottom: 0;
+  line-height: 1.3;
+  text-align: center;
+  font-size: 250%;
+  background-color: #ffaeda;
+  border-radius: 5px;
+  width: 50%;
+  color: #3E4659;
+}
+.content-subtitle {
+  text-align: center;
+  color: #01b2ac;
+  font-size: 190%;
+  padding: 1% 0%;
+  font-weight: 600;
+}
+.type1 { background-color: #ffaeda; color: #3E4659; }
```

Comment:
we use different pink so I'd suggest using `$litg-pink`

Reply:
Ok, I'll put all pink colors as litg-pink.

In my opinion this pink is already quite clear, and white text makes it difficult to read (more difficult than dark text). 
I use this site to see this colors details:
http://www.colorhexa.com/f29ead

Reply:
@LucasAndrad we're using this $litg-pink all over the page (you see more stuff when you log in) and it's also similar to the one in the header image so we don't want to change it for now

Reply:
No problem, litg-pink is just fine, I was suggesting to change just the text color, because in my opinion a darkest color is better to read than the white. What do you think?